mangaki/mangaki/templates/base.html

```diff
@@ -120,23 +120,24 @@
         </nav>
     </header>
 
+
     <div class="container">
-        {% block anonymous_rating_banner %}
-	  <div class="col-xs-12">
-            {% if request|has_anonymous_ratings %}
+        {% if request|has_anonymous_ratings %}
```

Comment:
I made that modification since there was an empty `div` on the top of the page when logged in.
I can revert it back if you wish 👍